ocaml/xenopsd/scripts/qemu-wrapper

```diff
@@ -327,7 +326,7 @@ def main(argv):
 
     qemu = subprocess.Popen(qemu_args, executable=qemu_dm, env=qemu_env,
                             stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
-                            preexec_fn=prepare_exec)
+                            preexec_fn=prepare_exec, pass_fds=open_fds)
```

Comment:
I can't see how the above `subprocess.call([clipboardd, ...)` works after the Py3 conversion. It expects an open fd but it has neither `close_fds=False` or `pass_fds=...`.

In any case, I think it would be preferable to use the `pass_fds` option in the call to xs-clipboardd and get rid of the preexec_fn option.

Reply:
Thanks for the review.
Code updated!